ue4cli/UE4BuildInterrogator.py

```diff
@@ -166,7 +166,10 @@ def _getThirdPartyLibs(self, platformIdentifier, configuration):
 		# (Ensure we always perform sentinel file cleanup even when errors occur)
 		try:
 			args = ['-Mode=JsonExport', '-OutputFile=' +jsonFile ] if self.engineVersion['MinorVersion'] >= 22 else ['-gather', '-jsonexport=' + jsonFile, '-SkipBuild']
```

Comment:
This is needed for `ue4 libs` to work with UE5:

```suggestion
			args = ['-Mode=JsonExport', '-OutputFile=' +jsonFile ] if (self.engineVersion['MajorVersion'] >= 5 or self.engineVersion['MinorVersion'] >= 22) else ['-gather', '-jsonexport=' + jsonFile, '-SkipBuild']
```